README.md

```diff
@@ -58,14 +58,29 @@ This microservice allows sending and receiving messages via FIX protocol
 Mangler is configured by specifying a list of transformations which it will try to apply to outgoing messages.   
 Each transformation has a list of conditions which message must meet for transformation actions to be applied.
 
-Condition is basically a field value check:
+Condition can be one of the following:
 
-```yaml
-tag: 35
-matches: (8|D)
-```
+1. field selector:
+
+   ```yaml
+    tag: 35
+    matches: (8|D)
+   ```
+
+   Where `tag` is a field tag to match and `matches` is a regex pattern for the field value.
+
+2. group selector:
 
-Where `tag` is a field tag to match and `matches` is a regex pattern for the field value.
+    ```yaml
+      group: group-name
+      contains:
+        - tag: 100
+          matching: A
+        - tag: 101
+          matching: C
+    ```
+
+   Where `group` is a name of a predefined group and `contains` is a list of field selectors
```

Comment:
Could you please explain what will be happened when a message contains more than one group matched by filter